test/misc/L10NTest.ts

```diff
@@ -23,6 +23,13 @@ export function L10NTest() {
         }
       });
     });
+    afterEach(() => {
+     String.toLocaleString({
+       'en': {
+         'Foo': 'Foo'
```

Comment:
Could not find a way to remove a definition from the locale, so I simply set it back to what it would return if there was no definition for it. This resolved the interference between tests were Foo was being translated to Bar.